src/plugins/drawing/components/vector-palette.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { VectorTypeButton } from "./vector-type-button";
+import { ToolbarSettings, VectorEndShape } from "../model/drawing-basic-types";
+
+export enum VectorType {
+  line = "line",
+  singleArrow = "arrow",
+  doubleArrow ="doublearrow"
+}
```

Comment:
OK, that makes sense.

While here I also removed kVectorTypes since it seems unnecessary - using Object.values(VectorType) instead. Unless there's a reason not to?

Reply:
The only reason I can think of for keeping `kVectorTypes` would be if the order is important, since I think `Object.values(VectorType)` might return them in an arbitrary (but probably consistent) order.